File: confirmar_login.py
#!C:\\Python26\\python.exe
#-*- coding: ISO-8859-1 -*-
import cgi, Cookie
print "Content-type: text/html; ISO-8859-1"
form = cgi.FieldStorage()
user = form.getvalue("username")
senha = form.getvalue("senha")
cookie = Cookie.SimpleCookie()

import MySQLdb
con = MySQLdb.connect('localhost', 'root', '') # faz a conexao
con.select_db('pythonweb')# seleciona o banco de dados
cursor = con.cursor()#prepara um objeto cursor
sql = "SELECT * FROM user WHERE nome = '%s' and senha = '%s'" % (user, senha)
print '''Content-type: text/html

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html><head>
<title>login</title>
</head>
<body>
'''
try:
	cursor.execute(sql)
	Results = cursor.fetchall()
	total = len(Results)
	if (total>0):
		print 'Set-Cookie: meucookie=%s'%(user)
		cookie['meucookie']= user
		print cookie
		print "<pre>Senha correta! você será redirecionado... <meta http-equiv='refresh' content='2;URL=paginauser.py'></pre></body></html>"
	else:	
		print "<pre>Senha errada! Não foi possivel o Login, você será redirecionado... <meta http-equiv='refresh' content='2;URL=user.py'></pre></body></html>"
except MySQLdb.Error, e:
    print "<pre>%s</pre></body></html>" % e
    exit(1)

File: confirmar_login.php
<?php
$username = $_POST['username'];
$senha = $_POST['senha'];
$erro="";
setcookie("username",$username); setcookie("senha",$senha);
echo "<font face=verdana size=1>";
$conexao=mysql_connect ("localhost", "root", "") or die ('I cannot connect to the database because: ' . mysql_error());
mysql_select_db ("pythonweb") or die("não foi possivel");
$sql = "SELECT * FROM user where user='$username';";
$resultado = mysql_query($sql, $conexao);
$linhas = mysql_num_rows($resultado);
if($linhas==0)
	{
		$erro="Usuário não encontrado! $username";
		$erro="<BR><BR><BR><BR><p align=\"center\">Usuário <b>não</b> encontrado <b>Aguarde...</b></p><meta http-equiv='refresh' content='1;URL=index.py'>";
	}
else
	{
		if($senha!=mysql_result($resultado,0,"senha"))
		{
			$erro ="Senha está incorreta!<br>";
			$erro .="Não foi possivel o Login <meta http-equiv='refresh' content='2;URL='user.py'>";
		}		
		else if($senha==mysql_result($resultado,0,"senha"))
		{
			echo "<BR><BR><BR><BR><BR><p align=\"center\">Login Efetuado ! <b>Aguarde....</b></p><meta http-equiv='refresh' content='2;URL=paginauser.py'>";
		}
		else
		{
			$erro="erro: Contate o Desenvolvedor";
			$erro="<p aling=center><a href=index.py>Voltar</a></p>";
		}
	}	

if(!empty($erro)){
	echo $erro;
	}
mysql_close($conexao);
?>
